pkg/persistentvolumeclaim/cstor.go

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright 2020-2021 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package persistentvolumeclaim
+
+import (
+	"fmt"
+	"time"
+
+	cstortypes "github.com/openebs/api/v2/pkg/apis/types"
+	"github.com/openebs/openebsctl/pkg/client"
+	"github.com/openebs/openebsctl/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/printers"
+)
+
+const (
+	cstorPvcInfoTemplate = `
+{{.Name}} Details :
+------------------
+NAME             : {{.Name}}
+NAMESPACE        : {{.Namespace}}
+CAS TYPE         : {{.CasType}}
+BOUND VOLUME     : {{.BoundVolume}}
+ATTACHED TO NODE : {{.AttachedToNode}}
+POOL             : {{.Pool}}
+STORAGE CLASS    : {{.StorageClassName}}
+SIZE             : {{.Size}}
+USED             : {{.Used}}
+CV STATUS	 : {{.CVStatus}}
```

Comment:
nit: is this indentation wrong?

Reply:
This is intended, without this the content gets shifted, not sure why.

```
mongo Details :
------------------
NAME             : mongo
NAMESPACE        : default
CAS TYPE         : cstor
BOUND VOLUME     : pvc-b84f60ae-3f26-4110-a85d-bce7ec00dacc
ATTACHED TO NODE : minikube-1
POOL             : 
STORAGE CLASS    : common-storageclass
SIZE             : 
USED             : 1.1 GiB
CV STATUS	     :  Healthy  -----> Without the indentation
PV STATUS        : Released

```